androidsdk/src/main/java/com/wootric/androidsdk/objects/WootricCustomThankYou.java

```diff
@@ -49,8 +49,8 @@
     private boolean scoreInUrl;
     private boolean commentInUrl;
 
-    public String getTextForScore(int scoreValue) {
-        Score score = new Score(scoreValue);
+    public String getTextForScore(int scoreValue, String surveyType) {
```

Comment:
This needs to be passed since the user shouldn't initialize this class with the `surveyType`, we should get that from the server. That is also why the `scoreValue` is passed in almost all the methods.